tendrl/node_agent/flows/import_cluster/__init__.py

```diff
@@ -21,25 +22,23 @@ def run(self):
         integration_id = self.parameters['TendrlContext.integration_id']
         tendrl_ns.tendrl_context.integration_id = integration_id
         tendrl_ns.tendrl_context.save()
-        node_list = self.parameters['Node[]']
+        node_list = self.parameters['Node[]'].split(", ")
```

Comment:
The jobs are created as json structures. I don't see why we have comma separated lists. Lists should be arrays.